### PR TITLE
Instantiate game

### PR DESCRIPTION
**Pull request by alott1925**
Move loading roles, role assignment, and message sending to the game class. Then, create the game when sheriff command is given.

### CODE REVIEW

File: src/game.js [hunk 1]
```diff
@@ -17,4 +22,49 @@ class Game{
...
+        let roleList = [];
+
+        roleData.forEach(element => {
+            roleList.push(new Role(element.roleID, element.title, element.description, element.image, element.color));
+        });
+
+        return roleList;
+    }
+
+    assignPlayerRoles(users){
```
Comment (alott1925) [inline src/game.js:38]: Not sure if this should be called `assignRoles()` ,  `createPlayers()`, or `assignPlayerRoles()`

File: src/player.js [hunk 2]
```diff
@@ -1,22 +1,6 @@
-class Player{
-	constructor(person, role) {
-		this.Person = person;
+module.exports = class Player{
+	constructor(user, role) {
```
Comment (alott1925) [inline src/player.js:2]: Debated between `user`, `userInterface`, or `mention`.

File: commands/sheriff.js [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
-		if(message.mentions.users.size < 5){
-            return message.channel.send('Five players are needed for a game of sheriff');
-        }
+		// console.log('message', message);
+		// if(message.mentions.users.size < 5){
+        //     return message.channel.send('Five players are needed for a game of sheriff');
+        // }
+
+		// Username#1318
+		let creator = `${message.author.username}#${message.author.discriminator}`;
```
Comment (alott1925) [inline commands/sheriff.js:13]: There's probably a more straightforward way to get this.

File: commands/sheriff.js [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
+		// 07/16/2021 7:51:47 PM
+		let date = new Date(Date.now()).toLocaleDateString("en-US");
+		let time = new Date(Date.now()).toLocaleTimeString("en-US");
+		let createdOn = `${date} ${time}`;
```
Comment (alott1925) [inline commands/sheriff.js:17]: Same here. I have a bad experience with Date objects anyway. Would rather use [moment.js](https://momentjs.com/) for dates, but we also need to think about what we want for logging anyway.

File: src/game.js [hunk 5]
```diff
@@ -1,9 +1,14 @@
...
+const Player = require('../src/player.js');
+const Role = require('../src/role.js');
+module.exports = class Game{
+    constructor(creator, createdOn, users){
         this.Creator = creator;
         this.CreatedOn = createdOn;
-        this.Players = players;
+        this.Roles = this.loadRoles();
+        this.Players = this.assignPlayerRoles(users);
+        this.messagePlayers();
```
Comment (alott1925) [inline src/game.js:11]: Lines 9-11 could be moved to `sheriff.js` if we didn't want the `Game` class doing _too_ much.

File: src/test.js [hunk 6]
```diff
@@ -1 +1,15 @@
-// empty file for right now
```
Comment (alott1925) [inline src/test.js:1]: Probably need to work with @ThisGuy169 to fix, if this edition of the bot is used.

**Review by aderrick318 — APPROVED** [review]

File: commands/sheriff.js (hunk 3 above)
Comment (aderrick318) [reply commands/sheriff.js:13]: Can't we use message.author.tag?

File: src/game.js (hunk 1 above)
Comment (aderrick318) [reply src/game.js:38]: Might need to discuss this to see what makes sense

**Review by aderrick318 — APPROVED** [review]

**Pull request merged**